go/app/infra.go

```diff
@@ -21,6 +21,10 @@ type Item struct {
 	Image    string `db:"image" json:"image"`
 }
 
+var jsonData struct {
+	Items []Item `json:"items"`
+}
```

Comment:
```suggestion
type JsonData struct {
	Items []Item `json:"items"`
}
```

この定義ですが、型ではなく変数を定義しちゃってます。ここでは型を定義して、それぞれの関数内で以下のように変数定義をするといいと思います！

```golang
var jsonData JsonData
```